frontend/src/lib/components/Animation/Animation.scss

```diff
@@ -0,0 +1,26 @@
+.Animation {
+    width: 100%;
+    max-width: 300px;
+    // A correct aspect-ratio is be passed via a style prop. This is as a fallback.
+    aspect-ratio: 1 / 1;
+    overflow: hidden;
+    opacity: 1;
+    transition: 400ms ease opacity;
+
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+
+    &.Animation__hidden {
+        opacity: 0;
+    }
+
+    .Animation-player {
```

Comment:
Nit: This is some non-typical convention 🤔 I think it should be `&.Animation--hidden` (block modifier) and `.Animation__player` (element within block)

Reply:
Thanks!